src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -614,7 +632,6 @@ function TTYGViewCtrl(
      */
     const onAgentListChanged = (agents) => {
         $scope.agents = agents;
-        $scope.showAgents = !agents.isEmpty();
```

Comment:
Why this was removed?

Reply:
Because this will open the panel when there are agents, but in the new requirement, we want it to be closed.